staging/src/k8s.io/component-helpers/resource/helpers.go

```diff
@@ -148,18 +148,17 @@ func AggregateContainerRequests(pod *v1.Pod, opts PodResourcesOptions) v1.Resour
 		for i := range pod.Status.ContainerStatuses {
```

Comment:
nit: Add the length of the init containers when constructing the map on the previous line